otherlibs/unix/unix.mli

```diff
@@ -1464,15 +1464,15 @@ val getsockopt_error : file_descr -> error option
 val open_connection : sockaddr -> in_channel * out_channel
 (** Connect to a server at the given address.
    Return a pair of buffered channels connected to the server.
-   Remember to call {!Pervasives.flush} on the output channel at the right
```

Comment:
There is still a `Pervasives.flush` in `open_process_args_out` (and on the labels side).

Reply:
Fixed